scripts/build.sh

```diff
@@ -31,6 +31,9 @@ ubuntu_release=(lsb_release -rs)
 # shellcheck source=scripts/common.sh
 . "${__dir}/common.sh"
 
+# Build examples for v1 and v2
+for f in `find ./examples/ -name "main.go"`; do (cd ${f%\/*} && echo "Building $PWD" && go build) ; done
```

Comment:
Please use rather `dirname` instead cutting parent directory from file name.
Also I would consider if "` `" for marking command substitution depreciated. (Please use rather $(foo)

Reply:
fixed